src/mount.ts

```diff
@@ -37,7 +38,14 @@ interface MountingOptions {
   stubs?: Record<string, any>
 }
 
-export function mount(originalComponent: any, options?: MountingOptions) {
+export function mount<T extends any>(
+  originalComponent: any,
+  options?: MountingOptions
+): VueWrapper<any>
```

Comment:
thanks @cexbrayat I will give this a quick test tmr and get it merged.

While we are here, can we get typings in the `props` mounting options? I was able to do that in an experiment I built [prior to this, found here](https://github.com/lmiller1990/vue-testing-framework/blob/bd83c4de5474aceaf95285635fa38908fa682b5d/packages/testing-framework/framework/mount.ts#L17)

Eg then we could do

```js
mount(Foo, {
  props: { 
    bar: 'sadf' // <= can we get typesafety here? that would be cool as
  }
})
```

If this is difficult to accomplish don't worry about it too much for now, but I agree, we should provide as many typings as possible.

Reply:
eg: `component: new () => ComponentPublicInstance<P>` (or something)

Reply:
Sure, I'll try to look into it soon, and I'll open a dedicated PR 👌

Reply:
@lmiller1990 FYI I added some typings tests using tsd in #72 , improved the typing of vm some more in #73 and started working on properly typings the props options in #74